website/content/en/docs/olm-integration/testing-deployment.md

```diff
@@ -8,51 +8,120 @@ This document discusses the behavior of `operator-sdk <run|cleanup>` subcommands
 and assumes you are familiar with [OLM][olm], related terminology,
 and have read the SDK-OLM integration [design proposal][sdk-olm-design].
 
-Currently only the package manifests format is supported by `<run|cleanup>` subcommands. Bundle support is coming soon.
-
 **Note:** before continuing, please read the [caveats](#caveats) section below.
 
-## `operator-sdk <run|cleanup> packagemanifests` command overview
+## `operator-sdk run bundle` command overview
+`operator-sdk run bundle` assumes OLM is already installed and running on your
+cluster. It also assumes that your Operator has a valid [bundle][bundle-format].
+See the [creating a bundle][creating-bundle] guide for more information. See the
+[CLI overview][doc-cli-overview] for commands to work with an OLM installation
+and generate a bundle.
+
+Let's look at the configuration shared between `run bundle`, `run
+packagemanifests` and `cleanup`:
+
+- **kubeconfig**: the local path to a kubeconfig. This uses well-defined default
+  loading rules to load the config if empty.
+- **namespace**: the cluster namespace in which Operator resources are created.
+  This namespace must already exist in the cluster.
+- **timeout**: a time string dictating the maximum time that `run` can run. The
+  command will return an error if the timeout is exceeded.
+
+Let's look at the anatomy of the `run bundle` configuration model:
+
+- **bundle-image**: specifies the Operator bundle image, this is a
+  required parameter. The bundle image must be pullable.
+- **index-image**: specifies an index image in which to inject the given bundle.
```

Comment:
We should add **namespace** as an optional flag to this configuration model as well, and that it will default to KUBECONFIG context if not provided.

Reply:
Namespace is covered in the common section above.

Reply:
**namespace** is part of the common section above this. I added that it is optional and defaults to kubeconfig.